drivers/hmis/app/graphql/mutations/update_bed_nights.rb

```diff
@@ -0,0 +1,46 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class UpdateBedNights < BaseMutation
+    argument :enrollment_ids, [ID], required: true
+    argument :bed_night_date, GraphQL::Types::ISO8601Date, required: true
+    argument :action, Types::HmisSchema::Enums::BulkActionType, required: true
+
+    field :success, Boolean, null: true
+
+    def resolve(enrollment_ids:, bed_night_date:, action:)
+      enrollments = Hmis::Hud::Enrollment.viewable_by(current_user).where(id: enrollment_ids)
+      raise 'not found' unless enrollments.count == enrollment_ids.uniq.length
+      raise 'project mismatch' unless enrollments.map(&:project_id).uniq.length == 1
```

Comment:
Is there some way to check that the project_id is the right project_id?  This is probably fine, just seems like maybe we could know which project we're working with?

Reply:
yeah we can just pass it in as an input. I'll update that.